integrations/facebook-pixel/lib/index.js

```diff
@@ -26,6 +27,7 @@ var FacebookPixel = (module.exports = integration('Facebook Pixel')
   .option('traverse', false)
   .option('automaticConfiguration', true)
   .option('whitelistPiiProperties', [])
+  .option('blacklistPiiProperties', [])
```

Comment:
Does this mean we also have to add UI to have this setting in the Destination UI?

Reply:
Yeah it's a net new addition though so no need to do anything funky with a DB migration which is nice.